roles/pulp_webserver/tasks/main.yml

```diff
@@ -13,11 +13,23 @@
   tags:
     - always
 
-- import_tasks: nginx.yml
-  when: pulp_webserver_server == 'nginx'
+- debug:
+    msg: >
+      {{ (pulp_install_source == "packages") | ternary(pulp_python_interpreter, pulp_install_dir ~ "/bin/python") }}
 
-- import_tasks: apache.yml
-  when: pulp_webserver_server == 'apache'
+- name: Check installed plugins for nginx snippets
+  script:
+    cmd: check_snippet.py {{ item.key | regex_replace("-", "_") | quote }} {{ pulp_webserver_server }}.conf
+  args:
+    executable: '{{ (pulp_install_source == "packages") | ternary(pulp_python_interpreter, pulp_install_dir ~ "/bin/python") }}'
+  register: snippets
+  with_dict: '{{ pulp_install_plugins_normalized }}'
+  failed_when: >
+    snippets.stderr is search("env: 'python': No such file or directory")
+  changed_when: false
+  check_mode: false
+
+- import_tasks: "{{ pulp_webserver_server }}.yml"
```

Comment:
if I understood correctly, we will check for Nginx snippets even when apache is selected?

Reply:
The name is fixed, but the variable pulp_webserver_server is used in the task. I will fix the name of the task to use the variable in my new/refreshed commit.

Reply:
So I think we can remove it: https://github.com/pulp/pulp_installer/blob/master/roles/pulp_webserver/tasks/apache.yml#L45-L55

Reply:
My bad, I missed that!